docs/content/docs/guide/certs.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Custom certificates
+weight: 8
+---
+# Custom certificates
+
+If you need to configure Pipelines-as-Code with a Git repository that
+requires a privately signed or custom certificate to access, then you will
+need to expose the certificate to Pipelines-as-Code.
+
+## OpenShift
+
+If you have installed Pipelines-as-Code through the OpenShift Pipelines
+operator, then you will need to [add your custom certificate to the cluster via
+the Proxy object.](https://docs.openshift.com/container-platform/4.11/networking/configuring-a-custom-pki.html#nw-proxy-configure-object_configuring-a-custom-pki)
+The operator will expose the certificate in all OpenShift Pipelines
+components and workloads, including Pipelines-as-Code.
+
+## Kubernetes
+
+### Create a ConfigMap containing the certificate
+
+```shell
+kubectl create configmap git-repo-cert --from-file=git.crt=<path to ca.crt>
+```
+
+### Mount the ConfigMap in the pods
+
+Follow [this guide](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#add-configmap-data-to-a-volume)
+to mount the ConfigMap in all Pipelines-as-Code Deployments in the cluster.
```

Comment:
Do we require this change for webhook pod as well ?

Reply:
i'm not aware if the webhook pod needs to communicate with the git repo 🤔

Reply:
Just the controller and the watcher....

Reply:
yes controller and the watcher required

May be need to update the instruction

Reply:
fixed